.github/workflows/main.yml

```diff
@@ -228,11 +228,11 @@ jobs:
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
     runs-on: ubuntu-16.04
     steps:
-    - name: install lz4
+    - name: Install conntrack
       shell: bash
       run: |
         sudo apt-get update -qq
```

Comment:
no need to do the update again. since it was done in last step and this takes time.

Reply:
Done.